flink-table/flink-table-api-scala/src/main/scala/org/apache/flink/table/api/expressionDsl.scala

```diff
@@ -1527,6 +874,20 @@ trait ImplicitExpressionConversions {
     * e.g. withoutColumns('b to 'c) or withoutColumns('c)
     */
   def withoutColumns(head: Expression, tail: Expression*): Expression = {
-    unresolvedCall(WITHOUT_COLUMNS, head +: tail: _*)
+    Expressions.withoutColumns(head, tail: _*)
+  }
+
+  /**
+   * Boolean AND in three-valued logic.
+   */
+  def and(predicate0: Expression, predicate1: Expression, predicates: Expression*): Expression = {
```

Comment:
How about other infix notation expressions for the Scala API? `lit()`, `call()` should also be present here. We should synchronize with `Expressions`? And add a comment that contributors should synchronize in the future.